### PR TITLE
README.md, RTD, Contributing.md & Code of Conduct

### PR DESCRIPTION
**Pull request by leeping-ng**

### CODE REVIEW

File: docs/source/index.md [hunk 1]
```diff
@@ -27,8 +27,21 @@ A **pipeline** governs the behavior of a chain of nodes. The diagram below shows
 <img src="https://raw.githubusercontent.com/aimakerspace/PeekingDuck/dev/diagrams/yolo_demo.drawio.svg">
 
 
+## Explore PeekingDuck's Features
 
-## Explore PeekingDuck Nodes
+The rest of this webpage contains more details about using PeekingDuck, including information on:
+- [Changing PeekingDuck nodes](getting_started/02_configure_pkdk) and their settings
+- [Full list](peekingduck.pipeline.nodes) of all PeekingDuck nodes, describing their behaviour, inputs, outputs and settings
```
Comment (sidney-tio) [inline docs/source/index.md:34]:
```suggestion
- [Official Documentation](peekingduck.pipeline.nodes) of all PeekingDuck nodes, describing their behaviour, inputs, outputs and settings
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
 # Contributing to PeekingDuck
```
Comment (sidney-tio) [inline CONTRIBUTING.md:2]: Add a ##Table of Content?

Comment (leeping-ng) [reply CONTRIBUTING.md:2]: Added.

File: docs/source/index.md (hunk 1 above)
Comment (leeping-ng) [reply docs/source/index.md]: Updated in both RTD and README.md

**Pull request merged**